community/kernel/src/main/java/org/neo4j/unsafe/impl/batchimport/staging/ForkedProcessorStep.java

```diff
@@ -252,6 +266,7 @@ public void run()
     {
         private final int id;
         private Unit current;
+        private volatile boolean end;
```

Comment:
Good catch, this is leftovers from when we shut down threads, now we only let them idle instead in order to avoid racing with the creation of new threads.